RNTester/e2e/__tests__/ActionSheetIOS-test.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails oncall+react_native
+ * @format
+ */
+
+/* global device, element, by, expect */
+const {
+  openComponentWithLabel,
+  openExampleWithTitle,
+} = require('../e2e-helpers');
+
+describe('ActionSheetIOS', () => {
+  beforeAll(async () => {
+    await device.reloadReactNative();
+    await openComponentWithLabel(
+      'ActionSheetIOS',
+      "ActionSheetIOS Interface to show iOS' action sheets",
+    );
+  });
+
+  it(
```

Comment:
The implementation of this test looks great! I think we could improve the readability and debuggability of this test by splitting it into a few separate test cases, and improving the test case description somewhat.

A good rule of thumb for test cases is that if it features the word "and", it should probably be split into two. This test has 3 ands, so it could maybe be three tests:
- should display options for a simple action sheet
- should show selection option when an option is tapped
- should dismiss when screen is tapped outside the action sheet

In addition to splitting on ands, I've made the language of the test cases more specific. It's not useful to use the word "correct" in test case names, since I can't think of a case where the expected outcome of the test would be incorrect options 😃 

Additionally, the convention is to spell `should` with a lowercase `s`, because the test runner will display them in a sentence format `{describe block} {it block}`.

OTOH I can see the tradeoff here that in the case of Detox tests, every test case adds a bit of overhead for reloading the bundle and navigating back to the screen, and we want the tests to be quick so too granular tests probably aren't good either.

IMO 3 would be a good number here. Thoughts @AnshG714?